actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -89,6 +89,9 @@ class ActionCable.Connection
         when message_types.welcome
           @monitor.recordConnect()
           @subscriptions.reload()
+        when message_types.unauthorized
+          ActionCable.log("Unauthorized. Stopping monitor and disconnecting.")
+          @close(allowReconnect: false)
```

Comment:
I think the reason [I was mentioned](https://github.com/rails/rails/pull/34194#issuecomment-431151836) may be because this would cause a significant conflict with my open pull request: https://github.com/rails/rails/pull/34177

I just addressed the last comment on the PR so I'm hoping that we'll be able to merge it soon. So, do we think it's reasonable to wait to merge this PR until that ES2015 conversion is merged?

Reply:
I'll happily rebase on master once #34177 is merged.

Reply:
I'm happy to help if you'd like